mod/deckorate_attribution/assets/style/reference.scss

```diff
@@ -74,3 +70,28 @@
 .copy-button {
   border-radius: 0rem 0.25rem 0.25rem 0rem;
 }
+
+.submit-button.btn.btn-primary {
+  align-items: center;
+  border-radius: 0rem 0.25rem 0.25rem 0rem;
+  display: flex; 
+  font-weight: bold;
+  margin-right: 0 !important
+}
+
+.nav-tabs {
+  display: flex;
+}
+
+.nav-item {
+  flex-grow: 1
+}
+
+.nav-link {
+  text-align: center
+}
+
+.nav-link:focus,
+.nav-link:hover {
+  background-color: #fff;
+}
```

Comment:
All of these are going to have unintended side effects..

Reply:
@ethn I am realizing that right now.